web/src/js/components/ContentView.jsx

```diff
@@ -33,10 +33,10 @@ function ContentView(props) {
         return <MetaViews.ContentTooLarge {...props} onClick={displayLarge}/>
     }
 
-    const View = ContentViews[contentView]
+    const View = ContentViews[contentView] || ContentViews['ViewServer']
```

Comment:
ContentViews[contentView] is undefined if that content view is an client contentview (like img, auto, or raw with the editor option) then i will use the "server content view". "server content view" is a view which uses the name of the active content view form the redux state and applies an request with the corresponding parameter.